tap_mysql/client.py

```diff
@@ -219,5 +438,10 @@ def get_records(self, context: dict | None) -> Iterable[dict[str, Any]]:
             if start_val:
                 query = query.filter(replication_key_col >= start_val)
 
-        for row in self.connector.connection.execute(query):
-            yield dict(row)
+        with self.connector._connect() as conn:  # noqa: SLF001
```

Comment:
We should probably change this in the SDK to be a public method instead of a private method.

https://github.com/MeltanoLabs/tap-postgres/blob/main/tap_postgres/client.py#L305 we do the same thing

Reply:
Would you mind creating an issue in the SDK?

Reply:
Done https://github.com/meltano/sdk/issues/2262